src/index.ts

```diff
@@ -146,13 +151,39 @@ export default class ServerlessConventions {
     // Check that the service name does not contain the word "service"
     if (serviceName.toLowerCase().includes('service')) {
       errors.push(
-        `Warning: Service name should not include the word "service"`
+        `Warning: Service name "${serviceName}" should not include the word "service"`
       );
     }
 
     // Check the length of the service name is not greater than x
     if (serviceName.length > 23) {
-      errors.push(`Warning: Service name must be less than 23 characters`);
+      errors.push(
+        `Warning: Service name "${serviceName}" must be less than 23 characters`
+      );
+    }
+
+    return errors;
+  }
+
+  // Stage name validation
+  // Must contains only lower case alphabet characters
+  // Must be only 3 characters
+  checkStageName(service: Service): Array<string> {
+    let errors: Array<string> = [];
+    const stageName = service.provider.stage;
+
```

Comment:
It seems impossible to me for `service.provider.stage` to be something else other than a valid string. I tried with `stage.provider.stage` as an empty string & Serverless automatically throw me an error . Their stage restriction is `^[a-zA-Z0-9-]+$`
![image](https://user-images.githubusercontent.com/108910975/192446856-758c1807-d249-436a-b6c2-27fd0c39ac26.png)